lib/jekyll-contentful-data-import/data_exporter.rb

```diff
@@ -34,8 +49,8 @@ def destination_directory
         File.join(base_directory, DATA_FOLDER, CONTENTFUL_FOLDER, SPACES_FOLDER)
       end
 
-      def destination_file
-        File.join(destination_directory, "#{name}.yaml")
+      def destination_file(file_name = name)
```

Comment:
You could change this to have the default as `path = ["#{name}.yaml"]` and then on the `File.join...` change it to `File.join(*[destination_directory].concat(path))` that way you could also have subpaths for content types with individual files if desired.

This will require a change on the `#destination_file` call to look like:

```ruby
grouped_entries.each do |content_type, entry_list|
  entry_list.each do |entry|
    File.open(destination_file([content_type, "#{entry[:id]}.yaml"]), 'w') do |file|
      file.write(::Jekyll::Contentful::Serializer.new(entries, config).serialize)
    end
  end
end
```

Or something along those lines